### ISSUE TITLE
DL3008 triggered when installing from a deb file

### ISSUE THREAD

**Issue by Rekhyt** [hadolint/hadolint#345]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Lint passes.

### Actual behavior
Lint fails with message
```DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`.```

### Steps to reproduce the behavior
Lint the below Dockerfile which installs a package from a .deb file. Specifying a version will make the lint pass but will make the apt-get command fail.

**Output of
  `docker run --rm hadolint/hadolint hadolint --version`:**
```bash
Haskell Dockerfile Linter v1.17.1-5-geac036cd
```

**Dockerfile** (if relevant)

```Dockerfile
FROM ubuntu:18.04

RUN apt-get update &&\
    wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.bionic_amd64.deb && \
    apt-get install --no-install-recommends -y ./wkhtmltox_0.12.5-1.bionic_amd64.deb && \
    apt-get autoremove -y &&\
    apt-get clean && rm -rf /var/lib/apt/lists/*
```

**Additional environment details (OS, stack version, etc.)**
